runtime/filesystem-http/src/main/java/io/aklivity/zilla/runtime/filesystem/http/internal/HttpPath.java

```diff
@@ -43,6 +43,7 @@ public final class HttpPath implements Path
 
     private final HttpFileSystem fs;
     private final URI location;
+    private final String authorization;
```

Comment:
Do we need to store the authorization on each `HttpPath`, or just read it from `HttpFileSystem` instead since we have a reference?